diff --git a/fftools/ffmpeg.c b/fftools/ffmpeg.c
--- a/fftools/ffmpeg.c
+++ b/fftools/ffmpeg.c
@@ -1079,7 +1079,6 @@ static void do_video_out(OutputFile *of,
 
     if (!ost->filters_script &&
         !ost->filters &&
-        (nb_filtergraphs == 0 || !filtergraphs[0]->graph_desc) &&
         next_picture &&
         ist &&
         lrintf(next_picture->pkt_duration * av_q2d(ist->st->time_base) / av_q2d(enc->time_base)) > 0) {
